subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -159,7 +159,9 @@ struct ticker_user_op_start {
 	uint32_t ticks_at_start;	/* Anchor ticks (absolute) */
 	uint32_t ticks_first;		/* Initial timeout ticks */
 	uint32_t ticks_periodic;	/* Ticker period ticks */
+#if defined(HAL_TICKER_REMAINDER_RANGE)
```

Comment:
I am thinking that `CONFIG_BT_TICKER_REMAINDER` should be renamed to `CONFIG_BT_TICKER_START_REMAINDER` as it is not meaning what it is....then it be maded depended on a new hidden Kconfig `CONFIG_BT_TICKER_REMAINDER_SUPPORT` . Then, this support can be turned off to save RAM/Flash.